kajiki/ir.py

```diff
@@ -6,13 +6,18 @@
 
 def generate_python(ir):
     cur_indent = 0
-    for node in flattener(ir):
+    for node in flattener(flattener(ir)):
```

Comment:
Could use a comment why we nest `flatterer` twice

Reply:
that was a stupid attempt